src/sync/Synchroniser.js

```diff
@@ -81,6 +84,39 @@ export class Synchroniser {
     this.batchSize = MIN_SYNC_BATCH_SIZE;
   };
 
+  fetchWrapper = async (url, opts) => {
+    const { body } = opts;
+
+    const bugsnagNotify = (message, response) => {
+      bugsnagClient.notify(new Error(message), {
+        context: 'SYNC ERROR',
+        metaData: { url, body, responseText: response.text() },
+      });
+    };
+    // eslint-disable-next-line no-undef
+    const response = await fetch(url, opts);
+    if (response.status < 200 || response.status >= 300) {
+      throw new Error('Connection failure while attempting to sync.');
+    }
+    let responseJson;
+    try {
+      responseJson = await response.json();
+    } catch (error) {
+      bugsnagNotify('Failed parsing response form server', response);
+      throw new Error('Unexpected response from sync server');
+    }
+
+    if (responseJson.error && responseJson.error !== '') {
```

Comment:
the `responseJson.error !== ''` is redundant

Reply:
19b0425ae4dc6ba3997e07d14e0e8154351caff8